scripts/kconfig/lxdialog/check-lxdialog.sh

```diff
@@ -2,11 +2,25 @@
 # SPDX-License-Identifier: GPL-2.0
 # Check ncurses compatibility
 
+# Android Q doesn't allow us to call pkg-config for some reason, so
+# stub out pkg-config to avoid build system errors. Uff...
+if [ -n "${PRODUCT_SOONG_NAMESPACES}" ] && [ -n "${TARGET_PRODUCT}" ]; then
+pkgconfig()
+{
+	:
+}
+else
+pkgconfig()
+{
+	pkg-config $@
```

Comment:
If you add command in front of running pkg-config you can just override pkg-config without renaming anything down the line. Or you just define this empty function when ever the above successedes with the name of pkg-config.